### PR TITLE
Batch netcore final release notes and minor fixes

### PR DESCRIPTION
**Pull request by matthchr**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [ ] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by matthchr** [comment]
@itowlson @tamram - Please review the changelog update
@markcowl @cormacpayne - Please hold off merging until either Ivan or Tamra sign off.

File: src/Batch/Client/changelog.md
Comment (itowlson) [inline src/Batch/Client/changelog.md]: Is this meant to communicate "the client library is now supported on .NET Core"?  If so perhaps say that up front, and talk about the physical aspects as a way of explaining how and which versions.  .NET nomenclature is so confusing that talking about `netstandard1.5` is about as meaningful (at least to _this_ developer) as writing our change logs in Ancient Greek!

**Review by itowlson — DISMISSED** [review]

**Comment by markcowl** [comment]
@itowlson @matthchr Waiting for signoff before merge

File: src/Batch/Client/changelog.md
Comment (tamram) [inline src/Batch/Client/changelog.md]: I'm a little confused by the last sentence. We're on 6.0.0 now, right?

Comment (tamram) [inline src/Batch/Client/changelog.md]: For the first two sentences, how about:

The client library is now supported on .NET Core. The package now includes a `netstandard1.5` assembly in addition to the `net45` assembly.

File: src/Batch/Client/changelog.md
Comment (tamram) [inline src/Batch/Client/changelog.md]: Looks great

File: src/Batch/Client/changelog.md
Comment (matthchr) [inline src/Batch/Client/changelog.md]: @tamram Updated to use your wording. We are on 6.0.0 now and we are releasing 6.1.0 (no changes except enabling netcore), so I am just telling people that if they are using net45, 6.0.0 == 6.1.0. The only change is the new netcore stuff.

**Comment by tamram** [comment]
@matthchr - looks good

**Comment by matthchr** [comment]
@markcowl - Okay, we're ready to merge this whenever you get a chance.

**Pull request merged**
